app-multimedia/yesplaymusic/autobuild/prepare

```diff
@@ -0,0 +1,13 @@
+acbs_copy_git
+
+abinfo "unset PREFIX to fix nvm use"
+unset PREFIX
+
+abinfo "Installing nvm ..."
+abinfo "source /etc/profile.d/30-nvm.sh to add nvm binary to env ..."
+abinfo "FIXME: source nvm.sh runs in autobuild4 strict mode, so use || true to ignore the error."
```

Comment:
Make this a comment.

Reply:
Also, "nvm.sh does not source successfully with Autobuild4 as it traps all errors. use `|| true' here to work around it."